oauth.go

```diff
@@ -16,32 +16,34 @@ limitations under the License.
 package main
 
 import (
-	"encoding/json"
-	"errors"
-	"fmt"
-	"io/ioutil"
-	"net/http"
+	"context"
 	"strings"
 	"time"
 
 	"github.com/gambol99/go-oidc/jose"
-	"github.com/gambol99/go-oidc/oauth2"
 	"github.com/gambol99/go-oidc/oidc"
+
```

Comment:
Looks like a few references to the gambol99 forks here ^^

Reply:
@gambol99 that's intentional. The goal of this PR is to migration from `github.com/gambol99/go-oidc/oauth2` to `golang/x/oauth2` and later we migrate the other packages.

Today keycloak-proxy and go-oidc are completely tied to the JOSE implementation, so I decided to do it in small steps.

I already updated these changes based on your feedback. Thank you very much.

Reply:
hi @abstractj .. yep, your right .. i rebased and checked, it does appear to refresh the token.

```shell
1.5275125002257237e+09	debug	keycloak-proxy/middleware.go:323	access permitted to resource	{"access": "permitted", "email": "gambol99@gmail.com", "expires": 4.774277085, "resource": "/*"}
1.5275125079456906e+09	info	keycloak-proxy/middleware.go:140	accces token for user has expired, attemping to refresh the token	{"client_ip": "127.0.0.1:58154", "email": "gambol99@gmail.com"}
1.527512508070129e+09	info	keycloak-proxy/middleware.go:176	injecting the refreshed access token cookie	{"client_ip": "127.0.0.1:58154", "cookie_name": "kc-access", "email": "gambol99@gmail.com", "expires_in": 59.929872189}
```